### ISSUE TITLE
Update Security Statement to mention ToU interventions

### ISSUE THREAD

**Issue by yaelberger-commits** [cds-snc/notification-planning#1646]
Update Security Statement to mention ToU interventions

- [x] Yedida to draft new content in EN
- [x] Yael and Melissa to review content
- [x] Marie Sophie to translate content 
- [x] Yedida to update GC Articles pages in EN and FR 
- [x] Yedida to clear cache
- [x] Yael QA in production that changes are accurate

**Comment by YedidaZalik** [comment]
There are several new TOU interventions we could reference in the Security statement. 
@mtoutloff should we reference all or just some of these:
- Prompting users to read through TOU on sign up and accept summary terms each time they sign in
- Misuse includes creating duplicate services or services incompatible with your original purpose.
- Create your GC Notify account with a work email address only you can access. Do not share inboxes, account information or passwords, even among staff.
- Secure your API keys and follow our rules for API calls and unprocessed requests.

(Also notice typo in TOU, under "Accept risks when sending text messages" in second bullet "ofinformation" needs a space

**Comment by YedidaZalik** [comment]
@mtoutloff recommends waiting for Yael's return to work on this card and I agree.
From Melissa:
I had a look at the Security statement this morning and I am still not clear on what was meant by "update security statement to mention TOU interventions". As we discussed yesterday, it's not clear if it means adding a statement that users are required to accept the TOU requirements upon login or to list the TOU interventions from the login screen in the Security statement.
My thinking is it's likely some type of statement regarding the former (that users are required to accept the TOU), here's why:
While the Security statement is a mix of both our responsibilities and our user's responsibilities (as well as suggestions to users to enhance security on their end), I would be reluctant to repeat what's already in the TOU, mostly because I think it might be impractical. Every time we update the TOU, we'd have to also review the Security statement to check if it also needs updates. And I'm not sure from a content perspective if it makes sense to have the same information appear in two different policy statements.
This also made me think whether we might want to consider moving some of the user responsibility content in the Security statement (under the headers "If you suspect a security breach or discover a vulnerability" and "You have security responsibilities") to the TOU.  I think it's more helpful for our users to have a clear delineation in our policy statements of what's their responsibility and what's ours and for them to all be in one place. 

From me: If we move content to the TOU, we should try hard not to add any new headings, It's our use of the headings that still keeps it scannable and allows for the shortened version on log in.

**Comment by andrewleith** [comment]
- Looking to clarify with @yaelberger-commits around intent behind this

**Comment by yaelberger-commits** [comment]
I agree with Melissa. The intent of updating our Security Statement was to highlight that we have added new interventions to make it easier for clients to read and understand the ToU, which in turn makes our product more secure by helping ensure clients follow the terms. 
I agree we don't need to repeat content in the Security Statement that's already in the Terms of Use or hgihlight "what's new."

**Comment by yaelberger-commits** [comment]
Also, if we find that it doesn't add value to users to mention these interventions in the Security Statement, we can decide not to mention this, but we did hear in research and testing that security advisors etc liked to see these extra steps we have taken

**Comment by YedidaZalik** [comment]
Yael indicated we only need to add to Security statement Prompting users to read through TOU on sign up and accept summary terms each time they sign in
[Slack thread ](https://gcdigital.slack.com/archives/C04TQMJCXEW/p1730738631074619)

**Comment by YedidaZalik** [comment]
Using suggest function, added proposed new content into Security statement in [GC Notify policy documents](https://docs.google.com/document/d/1UTrvyGwYNBCUGq55UADa6mg1V3F-hcT6ddi0uIjZEuw/edit?tab=t.0)

Will tag Melissa for review

**Comment by mtoutloff** [comment]
Reviewed and provided a couple comments

**Comment by YedidaZalik** [comment]
Implemented Melissa's feedback and she approved changes

**Comment by YedidaZalik** [comment]
Yael approved. 
Prepped doc for localization and sent to Marie-Sophie: https://docs.google.com/document/d/1exiQH0lQSzABwMgb5aNIgjX5PjT0VllGu4Nzno7WJgI/edit?tab=t.0#heading=h.cak2vui1yeeg

**Comment by YedidaZalik** [comment]
Localization is back! https://docs.google.com/document/d/1exiQH0lQSzABwMgb5aNIgjX5PjT0VllGu4Nzno7WJgI/edit?tab=t.0
Moving back to in progress

**Comment by YedidaZalik** [comment]
Discussed at stand-up
This is a small change so don't need to do as new page and archived page
